backend/gameplay/consumers.py

```
@@ -314,8 +350,45 @@ async def update_halfmove_clock(self, move_type, piece_type, chess_game_model: C
 			await self.update_game_attribute(chess_game_model, "halfmove_clock", chess_game_model.halfmove_clock + 1, should_save=False)
 
+	async def update_captured_material(self, board_placement, move_info: dict, en_passant_target_square, chess_game_model: ChessGame):
+		if not get_is_capture(board_placement, en_passant_target_square, move_info):
+			return
+		
+		destination_square = move_info["destination_square"]
+		piece_color_moved = move_info["piece_color"]
+		piece_type_moved = move_info["piece_type"]
+
+		if en_passant_target_square and piece_type_moved.lower() == "pawn" and str(destination_square) == str(en_passant_target_square) :
+			pawn_square = get_pawn_square_from_en_passant_square(en_passant_target_square, piece_color_moved)
```

Ruff (0.11.9)

`get_pawn_square_from_en_passant_square` may be undefined, or defined from star imports

(F405)